CONTRIBUTING.md

```diff
@@ -113,6 +113,54 @@ yarn lint:fix
 yarn build
 ```
 
+## Setting up your Developer Bot
+
+Head to the [Discord Developer Portal](https://discord.com/developers/applications), and create a new application (call it something like "AO Dev Test"), and optionally a testing server in Discord.
+
+In the portal, click "Bot" and add a bot. You'll need to enable Server Members Intent, and Message Content Intent like so:
```

Comment:
Make this in the following format:

Enable the following intents:
- Server members
- Message content

It makes it easier to skim